build.ps1

```diff
@@ -60,6 +60,18 @@ function NugetPack {
     }
 }
 
+function PackSelfContainProject {
```

Comment:
A bit confusing with NugetPack... which also pack self contain project

---
In reply to: [95931418](https://github.com/dotnet/docfx/pull/1167#discussion_r95931418) [](ancestors = 95931418)